docs/history.rst

```diff
@@ -5,4 +5,4 @@
 History
 *******
 
-.. include:: ../CHANGES.rst
+.. include:: ../NEWS (links).rst
```

Comment:
**suggestion** Unless I'm missing something, this link seems to be broken. This should fix it:

```suggestion
.. include:: ../NEWS.rst
```